python/housinginsights/sources/cama.py

```diff
@@ -0,0 +1,208 @@
+    Use public methods to retrieve data.
+    """
+
+    BASEURL = 'https://opendata.arcgis.com/datasets'
+    def __init__(self):
+        super().__init__(CamaApiConn.BASEURL)
+
+    def get_data(self):
+        """
+        Grabs data from CAMA. Individual CAMA property retrieves zone_type data
+        from MAR api. Count number of housing units and bedroom units per zone.
+        Return the count data (in dictionary form) to be processed into csv
+        by get_csv() method.
+        """
+
+        mar_api = MarApiConn()
+        result = self.get(urlpath='/c5fb3fbe4c694a59a6eef7bf5f8bc49a_25.geojson', params=None)
+
+        if result.status_code != 200:
+            err = "An error occurred during request: status {0}"
+            raise Exception(err.format(result.status_code))
+        cama_data = result.json()
+
+        """
+        Example of: anc_count = [OrderedDict([('zone_type', 'anc'), ('zone', 'ANC 2B'), 
+                                ('housing_unit_count', 10), ('bedroom_unit_count', 10)], etc)]
+        """
+        zone_types = ['anc', 'census_tract', 'neighborhood_cluster', 'ward', 'zip']
+        anc_count = []
+        census_count = []
+        cluster_count = []
+        ward_count = []
+        zipcode_count = []
+
+
+        """
+        Take each CAMA property data and retrieve the MAR data.
+        """
+        """
+        Certain square values have four digits + a letter. (ex. 8888E)
+        Square would be the first four digits and suffix would be the letter.
+        SSL sometimes comes as 8 digit string without spacing in the middle.
+        """
+        """
+        CAMA data includes bldgs under construction. CAMA's data includes AYB of 2018
+        as of June 2017. We eliminate all data points that are under construction and
+        don't provide any housing units and bedrm at this time.
+        """
+        for row in cama_data['features']:
+            try:
+                current_year = int(datetime.date.today().strftime('%Y'))
+                #Skipping none values for units under construction
+                if row['properties']['AYB'] is not None and int(row['properties']['AYB']) > current_year:
+                    continue
+
+                objectid = row['properties']['OBJECTID']
+                if len(row['properties']['SSL']) == 8:
+                    square = row['properties']['SSL'][:4]
+                    lot = row['properties']['SSL'][4:]
+                else:
+                    square, lot = row['properties']['SSL'].split()
+                suffix = ' '
+                if len(square) > 4:
+                    square = square[:4]
+                    suffix = square[-1]
+
+                mar_return = mar_api.get_data(square, lot, suffix)
+
+                ''' Count the housing units and bedrooms '''
+                num_units = 0
+                if row['properties']['NUM_UNITS']: num_units = row['properties']['NUM_UNITS']
+                if num_units == 0:
```

Comment:
@HLKwon Great, thanks for the information! 

Can you add a new file to `/docs/data/cama.md` where we will add documentation for this data set, and then add these comments into that file? We'll want to have this written up in our public-facing documentation in case anyone has questions. 

In the code, this can then just have a quick note that says 'see docs/data/cama.md for information on assumptions`